### ISSUE TITLE
"Unknown Flag: -_."

### ISSUE THREAD

**Issue by girvo** [lukeed/taskr#69]
```
➜  code git:(feature/fly-browserify) ✗ fly clear
[10:07:47] Unknown Flag: -_. Run fly -h to see the options.
```

With this Flyfile:

``` javascript
export default async function() {
    await this.log("Hello, world!");
}

export async function clear() {
    await this.clear('build/js/');
}
```

Any ideas what would be causing that? I'm running Fly 0.4.2

```
➜  code git:(feature/fly-browserify) ✗ fly -v
fly, 0.4.2
```

**Comment by MadcapJake** [comment]
I'm getting this too, looks like specifying tasks on the command line is broken.

**Comment by ghost** [comment]
Confirmed. This is a bug in Parsec where it's detecting the task names as invalid flags. If you 

```
...
.parse(process.argv, { strictMode: false })
```

with `strictMode` set to `false` it does work.

Basically I forgot to the detect the end of options. Fixing this ASAP.

**Comment by ghost** [comment]
@girvo @MadcapJake Thanks. I just [fixed this](https://github.com/bucaran/parsec/issues/7) in Parsec land, will fix this here in Fly in the next release along with other small fixes and as much work I can add over the weekend.

**Comment by deepsweet** [comment]
hi @bucaran, is there any estimate on next release? getting this error too and can't finish gulp replacing :)

**Comment by ghost** [comment]
Yes, that would be sometime today with @0.5.0 → flyjs/fly/milestones/v0.5.0

Terribly sorry for the delay, but I was rewriting the CLI parser from [scratch](https://github.com/bucaran/parsec). And the good news is that I was able to go from 200 lines and a few deps to _no_ deps and only 50 loc (with still same set of features). Yay!

**Comment by deepsweet** [comment]
sounds great :fire:

**Comment by ghost** [comment]
:metal:

**Comment by ghost** [comment]
Fixed in [@v0.5.0](https://github.com/flyjs/fly/releases/tag/v0.5.0)
